packages/api-request-builder/src/build-query-string.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 import type {
   ServiceBuilderDefaultParams,
-} from 'types/sdk'
+} from '../../../types/sdk'
```

Comment:
Still having the errors straight from a fresh git clone on master.
More like a warning though as everything still "seems" to work fine as it is...